src-tauri/src/utils.rs

```diff
@@ -9,6 +10,27 @@ pub(crate) struct DiskStats {
     total_bytes: u64,
 }
 
+#[tauri::command]
+pub(crate) fn open_folder(dir: String) {
+    #[cfg(target_os = "windows")]
+    Command::new("explorer")
+        .arg(dir)
+        .spawn()
+        .expect("could not open to specified directory");
```

Comment:
```suggestion
        .expect("could not open the specified directory");
```

_very_ minor nitpick feel free to ignore 😄

Reply:
lemme fix that in the other pr, thanks for noticing that!